manifests/input.yaml

```diff
@@ -4,8 +4,8 @@ version: 1.0
 work/input/JRA55do-ESMFmesh.nc:
   fullpath: /g/data/ik11/inputs/access-om3/0.x.0/share/meshes/JRA55do-ESMFmesh.nc
   hashes:
-    binhash: 5195a7517bd0905add0f97ceecc0503f
-    md5: 154215afc3b742e3da9ac77e4eac6813
+    binhash: 1147443c3fe237a480af1b4700159e21
```

Comment:
Why has this file changed?

Reply:
It changed when the mesh generation script got moved to the `om3-scripts` repo (all that changed is the metadata about how the file was generated - the contents are the same)

Reply:
Maybe that needs its own commit too :)